recipes/sphinx-jupyterbook-latex/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "sphinx-jupyterbook-latex" %}
+{% set pip_name = name.replace("-", "_") %}
+{% set version = "0.4.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ pip_name }}/{{ pip_name }}-{{ version }}.tar.gz
+  sha256: 06d6c15f5ee572d4852421fb9e94ed1dcfd0ff68e03eb0ba61c2829b8114ac8e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - sphinx >=3,<4
+    - myst-nb >=0.11,<0.13
+    - sqlalchemy >=1.3.12,<1.4
+
+test:
+  requires:
+    - pip
+  commands:
+    - pip check
+  imports:
+    - sphinx_jupyterbook_latex
+    - sphinx_jupyterbook_latex.theme
+
+
+about:
+  home: https://github.com/executablebooks/sphinx-jupyterbook-latex
+  license: BSD-3-Clause 
+  license_file: LICENSE
+  summary: |
+    Sphinx-jupyterbook-latex: Supporting LaTeX infrastructure for Jupyter Book
+  dev_url: https://github.com/executablebooks/sphinx-jupyterbook-latex
+
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
+    - AakashGfude
+    - conda-forge/jupyter-book
```

Comment:
Almost all are already members of @conda-forge/jupyter-book (for whom i hereby accept maintainership)... @mmcky  can you just drop a comment that you are interested in helping maintain this?

Reply:
@bollwyvl sorry for slow reply. Happy to help maintain this.